_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -22,19 +22,24 @@ When the primary shard sends a checkpoint to replica shards on a refresh, a new
 
 Segment replication is the first feature in a series of features designed to decouple reads and writes in order to lower compute costs.
 
+**Limitation:** Segment replication is not compatible with the [Alerting]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/) plugin. The Alerting plugin relies on the latest data being available on replica shards when using the `immediate` refresh policy. The `immediate` refresh policy in segment replication can resource-intensive and introduce latency due to real-time data synchronization. As such, segment replication cannot guarantee the availability of the `immediate` refresh policy and thus lead to stale replica shards. This especially impacts [document-level monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/api/#document-level-monitors). Additionally, segment replication does not support the `wait_for` value in the `refresh` query parameter.   
```

Comment:
We also need to call out that Security Analytics is not compatible at all with Segment replication since it heavily uses Document Level monitors behind the scenes.

Reply:
Revised